src/hotspot/share/oops/instanceKlass.inline.hpp

```diff
@@ -45,6 +46,12 @@ inline int InstanceKlass::itable_offset_in_words() const { return start_of_itabl
 
 inline oop InstanceKlass::static_field_base_raw() { return java_mirror(); }
 
+inline Symbol* InstanceKlass::field_name        (int index) const { return field(index).name(constants()); }
+inline Symbol* InstanceKlass::field_signature   (int index) const { return field(index).signature(constants()); }
```

Comment:
There should not be spaces between a method name and the opening `(`. I'm really not a fine of this kind of alignment.